### PR TITLE
allow automation to create k8s 1.27 versions

### PR DESCRIPTION
**Pull request by laverya**
#### What this PR does / why we need it:

This allows us to add k8s 1.27.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by laverya** [comment]
The updated automation was run and produced https://github.com/replicatedhq/kURL/pull/4371

File: packages/kubernetes/template/script.sh
Comment (rrpolanco) [inline packages/kubernetes/template/script.sh:1]: We should remove the 1.23 and 1.22 version discovery logic.

**Review by emosbaugh — CHANGES_REQUESTED** [review]
can you separate testgrid full and deploy update into a different pr? im worried there will be some lag in supporting 1.27 and testgrid will fail until then

**Comment by laverya** [comment]
@emosbaugh yeah can do - should I leave in the minimal-127 test though?

File: packages/kubernetes/template/script.sh
Comment (laverya) [reply packages/kubernetes/template/script.sh:1]: Probably! For now though, it doesn't harm anything to keep it in

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**